modAionImpl/src/org/aion/zero/impl/db/AionRepositoryImpl.java

```diff
@@ -376,6 +376,18 @@ public ByteArrayWrapper getStorageValue(Address address, ByteArrayWrapper key) {
         return (details == null) ? EMPTY_BYTE_ARRAY : details.getCode(codeHash);
     }
 
+    @Override
+    public byte getVmType(Address contract) {
```

Comment:
Is it possible to return `TransactionTypes` instead here? Just so that the caller gets a more meaningful object in their hands?

Reply:
I will refactor the `TrsansactionTypes` into a enum since they are currently constants.